LongWarOfTheChosen/Config/XComLW_UI.ini

```diff
@@ -79,3 +79,7 @@ RECRUIT_FONT_SIZE_CTRL = 22					; Recruit screen font size for controller users.
 RECRUIT_Y_OFFSET_CTRL = -3					; Moves the stats up/down; negative numbers move up / positive numbers move down.
 RECRUIT_FONT_SIZE_MK = 20					; Recruit screen font size for mouse & keyboard users.
 RECRUIT_Y_OFFSET_MK = -2;					; Moves the stats up/down; negative numbers move up / positive numbers move down.
+
+[LW_Overhaul.UIScreenListener_SquadSelect_LW]
+HIDE_SAVESQUAD_BTN_WITH_CONTROLLER = false	; Hide the 'Save Squad' button which appears while viewing a Long War squad's soldiers.
+HIDE_SQUADCONTAINER_WITH_CONTROLLER = false	; Hide the 'Squad Container' which allows for squad selection on the Squad Select screen.
```

Comment:
```suggestion
```

Reply:
No problems with these changes - this works even better than config variables.
Also, like I said before, I'll look at integrating things in a little while when 'other controller tasks' die down.
Thanks.